log.c

```diff
@@ -78,8 +81,7 @@ void message(int level, const char *format, ...)
                 priority |= LOG_DEBUG;
                 break;
             case MESS_DEBUG:
-            case MESS_VERBOSE:
-            case MESS_NORMAL:
+            case MESS_WARN:
```

Comment:
Should `MESS_WARN` maybe translate to a syslog priority of `LOG_WARNING` (or `LOG_NOTICE`)?

Reply:
Yes, that makes perfect sense.